cluster/ci/origin/logging-configmap.yaml

```diff
@@ -22,6 +22,6 @@ objects:
         "marker-file": "/var/run/logging/marker",
         "gcs-bucket": "${GCS_BUCKET}",
         "gce-credentials-file": "/var/run/secrets/gce/gce.json",
-        "artifact-dir": "/var/run/logging/artifacts",
+        "artifact-dir": "/go/src/github.com/openshift/origin/_output/scripts/test-go/artifacts",
```

Comment:
should probably be OK to terminate at `scripts/`, below that is going to vary

Reply:
fixed